index.bs

```diff
@@ -744,7 +754,10 @@ To <dfn>populate the pose</dfn> of a {{XRSpace}} |sourceSpace| in an  {{XRSpace}
   1. Let |session| be |frame|'s {{XRFrame/session}} object.
   1. If |sourceSpace|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
   1. If |destinationSpace|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
-  1. If |destinationSpace|'s pose cannot be determined relative to |sourceSpace| at the time represented by |frame|, set |pose| to <code>null</code>.
+  1. If |destinationSpace|'s pose cannot be determined relative to |sourceSpace| at the time represented by |frame|, set |pose| to <code>null</code>, and abort these steps.
+  1. If either |sourceSpace| or |destinationSpace| have an [=XRSpace/internal type=] of {{XRReferenceSpaceType/identity}}, perform the following steps:
+      1. If the other space has am [=XRSpace/internal type=] that is {{XRReferenceSpaceType/identity}} or [=XRSpace/viewer space=], set |pose| to <code>null</code>, and abort these steps
+      1. If the other space has a [=XRSpace/internal type=] that is [=XRSpace/input target ray=], and its [=XRSpace/input source=] has a {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/screen}}, set |pose| to <code>null</code>, and abort these steps
```

Comment:
These steps are inverted. The logic should be that UNLESS the internal type is viewer space or target ray with a targetRayMode of screen, return `null`. Also, I'm not sure that an identity space should be relatable to a different identity space.

Reply:
Whoops.

I can remove the identity space thing and invert the logic